modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -4,7 +4,10 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: esb, deploy, configuration, cluster, ha, high availability, downtime
 
-A *cluster* is a set of Mule runtimes that acts as a unit. In other words, a cluster is a virtual server composed of multiple nodes. The nodes (Mule runtimes) in a cluster communicate and share information through a distributed shared memory grid. This means that the data is replicated across memory in different physical machines.
+[IMPORTANT]
+Clustering is not available in xref:runtime-manager::cloudhub.adoc[CloudHub], see xref:runtime-manager::cloudhub-fabric.adoc[CloudHub Fabric] for details of how workers can be shared or doubled to scale your application and provide high availability.
```

Comment:
I don't think these inline links to CloudHub and CloudHub Fabric are necessary here. 
At this point we assume the user is familiar with those products. 
In this particular case, I think these links would make more sense in the `See Also` section.